### DISCUSSION TITLE
Search chat history

### DISCUSSION THREAD

**Discussion by Moshyfawn** [zed-industries/zed#23913] [Feature Requests]
Summary: Add the ability to search the chat history in Zed.

_Probably another one of those more "management" items, but I go into why I think these are valid in my eyes in #10135._

> This is more of a community management type request, but as Zed is more than an IDE and strives to be a collaboration platform, there's a fair chance there will be more use cases where communities would like to see moderation tools and better chat management support UX.

<hr />

I would like to be able to search the chat history.

**Comment by maxdeviant** [comment]
I think this will make sense to pick up once we visit the chat functionality. Going to move to a discussion, in the meantime.
